asciidoc/modules/con_mg_changes-in-vertx-jdbc-client.adoc

```diff
@@ -1,8 +1,240 @@
 [id="changes-in-vertx-jdbc-client_{context}"]
 = Changes in {VertX} JDBC client
 
+From {VertX} {v4}, the JDBC client supports SQL client. The SQL common module has also been merged in JDBC client, that is, `io.vertx:vertx-sql-common` merged in `io.vertx:vertx-jdbc-client` module. You will have to remove the `io.vertx:vertx-sql-common` dependency file because `io.vertx:vertx-jdbc-client` will include it. With the merging of SQL common client, all the database APIs have been consolidated into the JDBC client.
+
+In {VertX} {v4}, the SQL client has been updated to include the following clients:XXX
+
```

Comment:
@pmlopes Does this read correct?

In {VertX} {v4}, the SQL client has been updated to include the following clients:

Reply:
yes